libvmi/events.h

```diff
@@ -46,6 +46,8 @@ extern "C" {
 
 #include <stdbool.h>
 
+#include "libvmi.h"
```

Comment:
No, you should not use `""` in a public header. It must be `<>`

Reply:
Is that part of some convention I am not aware of? With `""` the preprocessor will look in the current directory of the file first which is exactly where `libvmi.h` will be located. It's not like we are using some public header of another project.

Reply:
What are you talking about? This is a public header

Reply:
Let me phrase it differently: `libvmi.h` and `events.h` wil be deployed to the same directory (e.g. `/usr/include/libvmi`). If the preprocessor sees `#include "libvmi.h"` in `events.h` it will look for `libvmi.h` in the same directory as `events.h` before it will consider any other include rules. We are not dependent on any global include paths here, at least not to my knowledge. That being said, I don't mind using `<>` here although I think it would make more sense to use `""`. I've already changed it btw.